documents/src/pages/elements/tree-select.md

```diff
@@ -520,6 +520,48 @@ This control will save the current selection, and result of the firing of the `v
 
 This control will abort any current selection changes and go back to the tree state when first opened.
 
+## Manipulating item properties
+
+Item properties of Tree Select could be read and updated programmatically through its [TreeManager](./custom-components/utils/tree-manager) which is available as `treeManager` property. Retrieve [TreeNode(s)](./custom-components/utils/tree-node) representing each item by calling `getTreeNode()` or `getTreeNodes()` of `treeManager`. In addition, they could be used for [custom renderer](./custom-components/utils/tree-node#custom-renderer) too.
```

Comment:
Let's move "Custom Renderer" from "Tree Node" to "Tree" & "Tree Select" + revise it content a little as readers focus on Component first rather than "Tree Node" first.